k9mail-library/src/main/java/com/fsck/k9/mail/store/imap/QresyncResponse.java

```diff
@@ -0,0 +1,54 @@
+package com.fsck.k9.mail.store.imap;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fsck.k9.mail.MessagingException;
+
+
+public class QresyncResponse {
```

Comment:
there is no `QRESYNC` command. is this a response to a `SELECT` with `QRESYNC` parameter? if so, the name should reflect that